apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -70,10 +77,12 @@ class TeacherFeedback extends Component {
       url: `/api/v1/teacher_feedbacks/get_feedback_from_teacher?student_id=${this.state.studentId}&level_id=${this.props.serverLevelId}&teacher_id=${this.props.teacher}`,
       method: 'GET',
       contentType: 'application/json;charset=UTF-8',
-    }).done(data => {
+    }).done((data, textStatus, request) => {
       this.setState({latestFeedback: [data]});
+      this.setState({token: request.getResponseHeader('csrf-token')});
     }).fail((jqXhr, status) => {
       console.log(status + "  " + jqXhr.responseJSON);
```

Comment:
remove console.log , if it is no longer needed?

Reply:
Across our code base, we are inconsistent around sometimes printing a log or error to the console when an API fails. (cc @aoby, who pointed out the same thing) Is it our preferred style to not print a message?

Reply:
I've always viewed printing to console.log in these cases as a last resort when no other error handling is being performed. these won't show up as js errors in new relic / ui tests, and are useful only for developers running locally or if we ask a user to check the error console. because we're now showing a warning in the UI, I don't see any value added by keeping this console.log. @aoby or @islemaster feel free to disagree

Reply:
Agreed. I sometimes do it as a development aid, but don't leave it in the final code.